### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by Stoyvo** [tcarlsen/atom-csslint#11]
[Enter steps to reproduce below:]

1. At bottom of CSSLint Report it defines "Too many @font-face declarations(x); Click on "at line undefined" to the left of the report

**Atom Version**: 0.189.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: [csslint](https://github.com/tcarlsen/atom-csslint) package, v1.0.4

### Stack Trace

Uncaught TypeError: Cannot read property 'length' of undefined

```
At C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\node_modules\text-buffer\lib\text-buffer.js:399

TypeError: Cannot read property 'length' of undefined
  at TextBuffer.module.exports.TextBuffer.lineLengthForRow (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\node_modules\text-buffer\lib\text-buffer.js:399:29)
  at TextBuffer.module.exports.TextBuffer.clipPosition (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\node_modules\text-buffer\lib\text-buffer.js:953:53)
  at MarkerManager._results.push._this.(anonymous function) [as clipPosition] (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\node_modules\delegato\lib\delegator.js:67:61)
  at Marker._results.push._this.(anonymous function) [as clipPosition] (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\node_modules\delegato\lib\delegator.js:67:61)
  at Marker.module.exports.Marker.setHeadPosition (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\node_modules\text-buffer\lib\marker.js:202:23)
  at Marker.module.exports.Marker.setHeadBufferPosition (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\src\marker.js:210:32)
  at C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\src\cursor.js:129:31
  at Cursor.module.exports.Cursor.changePosition (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\src\cursor.js:684:7)
  at Cursor.module.exports.Cursor.setBufferPosition (C:\Users\stoyvo\AppData\Local\atom\app-0.189.0\resources\app\src\cursor.js:127:19)
  at [object Object].LineMessageView.goToLine (C:\Users\stoyvo\.atom\packages\csslint\node_modules\atom-message-panel\lib\LineMessageView.js:60:49)
  at HTMLDivElement.<anonymous> (C:\Users\stoyvo\.atom\packages\csslint\node_modules\atom-message-panel\node_modules\space-pen\lib\space-pen.js:214:36)
  at HTMLDivElement.jQuery.event.dispatch (C:\Users\stoyvo\.atom\packages\csslint\node_modules\atom-message-panel\node_modules\space-pen\vendor\jquery.js:4676:9)
  at HTMLDivElement.elemData.handle (C:\Users\stoyvo\.atom\packages\csslint\node_modules\atom-message-panel\node_modules\space-pen\vendor\jquery.js:4360:46)

```

### Commands

```
     -7:04.8 core:save (atom-text-editor.editor)
     -5:11.4 core:move-right (atom-text-editor.editor)
     -5:10.2 editor:newline (atom-text-editor.editor)
     -5:09.8 core:move-up (atom-text-editor.editor)
     -5:09.4 editor:toggle-line-comments (atom-text-editor.editor)
  3x -5:06.1 core:backspace (atom-text-editor.editor)
     -5:01.5 core:delete (atom-text-editor.editor)
     -4:57.3 core:save (atom-text-editor.editor)
  2x -3:21.6 core:backspace (atom-text-editor.editor)
     -3:18.2 core:save (div.am-panel.tool-panel.panel-bottom.native-key-bindings)
  2x -2:56.7 core:backspace (atom-text-editor.editor)
     -2:54.5 core:save (atom-text-editor.editor)
  2x -2:40.6 core:backspace (atom-text-editor.editor)
     -2:38.8 core:save (atom-text-editor.editor)
  4x -0:40.9 core:backspace (atom-text-editor.editor)
     -0:38.9 core:save (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "projectHome": "\\\\UBUNTU\\www",
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "git-diff",
      "language-c",
      "language-clojure",
      "language-csharp",
      "language-gfm",
      "language-git",
      "language-go",
      "language-java",
      "language-less",
      "language-make",
      "language-objective-c",
      "language-perl",
      "language-property-list",
      "language-python",
      "language-ruby",
      "language-ruby-on-rails",
      "language-sass",
      "language-shellscript",
      "language-toml",
      "language-yaml",
      "metrics",
      "autocomplete",
      "vim-mode"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v2.8.0
color-picker, v1.5.0
csscomb, v0.1.2
csslint, v1.0.4
editor-stats, v0.17.0
editorconfig, v0.3.3
jshint, v1.3.0
minimap, v4.7.5
tabs-to-spaces, v0.9.2

# Dev
No dev packages
```

**Comment by tcarlsen** [comment]
this is a little odd. it looks like your version of csslint havent updated its dependencies.

this was a error in amp but was fixed... just to sound like a pc support guy:

> can you try uninstall and reinstall the package and see if this error still occurs?

**Comment by tcarlsen** [comment]
hope taht this is fixed in `v1.1.2`
